homeassistant/components/zha/core/channels/base.py

```diff
@@ -155,13 +155,13 @@ def async_send_signal(self, signal: str, *args: Any) -> None:
 
     async def configure_reporting(
         self,
-        attr,
+        attr: int,
         report_config=(
```

Comment:
```suggestion
        report_config: Tuple[int, int, int] = (
```
Don't forget to `from typing import Tuple`